debian/control

```diff
@@ -0,0 +1,165 @@
+Source: letsencrypt
+Section: web
+Priority: optional
+Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
+Uploaders: Harlan Lieberman-Berg <hlieberman@setec.io>,
+	   Francois Marier <francois@debian.org>
+Build-Depends: debhelper (>= 9),
+	       dialog,
+               dh-python,
+               python-all,
+	       python-augeas,
+               python-configargparse,
+	       python-configobj,
+               python-dialog,
+               python-docutils,
+	       python-mock,
+	       python-ndg-httpsclient,
+               python-nose,
+               python-nosexcover,
+               python-openssl (>= 0.15),
+               python-parsedatetime,
+	       python-psutil,
+	       python-pyasn1,
+	       python-pyparsing,
+               python-repoze.sphinx.autointerface,
+	       python-requests,
+               python-rfc3339,
+	       python-setuptools,
+               python-sphinx,
+               python-sphinx-rtd-theme,
+       	       python-tz,
+               python-werkzeug,
+               python-zope.component,
+	       python-zope.interface
+Standards-Version: 3.9.6
+Homepage: https://www.letsencrypt.org/
+X-Python-Version: >= 2.7
+
+Package: python-acme
+Section: python
+Architecture: all
+Depends: openssl,
```

Comment:
The point is that someone has to maintain and sync `setup.py` with those lists - putting unnecessary stuff, such as `openssl`, makes this process harder.

If there is no dependency, then don't add it. If there is a transitive dependency (such as `ca-certificates`), but still used explicitly by our package - add it, so that if upstream changes their deps list we are not affected.